include/mbedtls/ctr_drbg.h

```diff
@@ -111,31 +132,32 @@ typedef struct
 mbedtls_ctr_drbg_context;
 
 /**
- * \brief               CTR_DRBG context initialization
- *                      Makes the context ready for mbedtls_ctr_drbg_seed() or
- *                      mbedtls_ctr_drbg_free().
+ * \brief               This function initializes the CTR_DRBG context,  
```

Comment:
Blocker: Please remove the trailing whitespace.

Reply:
fixed